Assets/Scripts/Game/Utility/PopulationManager.cs

```diff
@@ -200,6 +205,7 @@ void UpdateMobiles()
                     poolItem.npc.Motor.gameObject.SetActive(false);
                     poolItem.active = false;
                     poolItem.scheduleEnable = false;
+                    poolItem.npc.Billboard.transform.localPosition = Vector3.zero;
```

Comment:
This has been found to cause an exception after fast travelling around a while (it triggered right on arrival at a new town). I'll add a quick null check here, but wanted to let you know in case some other action is also needed. :)

Reply:
Thank you. I believe this happens if poolItem is marked for recycle before npc info and texture are set, which is also when position is changed. If this does't happens it shouldn't be needed to reset to parent position, so a null check should be enough.